mne/cov.py

```diff
@@ -386,66 +389,90 @@ def compute_raw_covariance(raw, tmin=None, tmax=None, tstep=0.2,
     picks : array-like of int | None (default None)
         Indices of channels to include (if None, all channels
         except bad channels are used).
+    keep_sample_mean : bool (default False)
```

Comment:
If the parameter is not exposed, then that means making it `keep_sample_mean=True` or `keep_sample_mean=False` always.

Always using `keep_sample_mean=True` will make getting the old behavior of `compute_raw_covariance`, and thus the behavior equivalent to the version computed with `mne_process_raw` in the sample dataset, impossible to get in mne-python.

Always using `keep_sample_mean=False` will make it impossible to match the behavior of `compute_covariance` if you just did `compute_covariance(Epochs(..., make_fixed_length_events(...), ...))` with no `keep_sample_mean` arg (because there it's default `True`).

Which one are you supporting?